libraries/Microsoft.Bot.Builder.Dialogs/Prompts/ChoicePrompt.cs

```diff
@@ -64,13 +64,14 @@ protected override async Task OnPromptAsync(ITurnContext turnContext, IDictionar
             var choices = options.Choices ?? new List<Choice>();
             var channelId = turnContext.Activity.ChannelId;
             var choiceOptions = ChoiceOptions ?? DefaultChoiceOptions[culture];
+            var choiceStyle = options.ChoiceStyle ?? Style;

```

Comment:
I would make the options name Style instead of ChoiceStyle so that it matches the property name.

Reply:
Done.